src/lib.rs

```diff
@@ -82,6 +82,8 @@ pub mod task;
 #[cfg(not(feature = "riscv-ulp-hal"))]
 pub mod timer;
 #[cfg(not(feature = "riscv-ulp-hal"))]
+pub mod touch;
```

Comment:
gate this against esp32 esp32s2 and esp32s3 simmilar to how the ulp is gated 2 lines under here